daemon/api.go

```diff
@@ -847,7 +794,7 @@ func getLogs(c *Command, r *http.Request) Response {
 
 	for i := range rawlogs {
 		logs[i] = map[string]interface{}{
-			"timestamp": rawlogs[i].RawTimestamp(),
```

Comment:
It appears that this is the only use of "RawTimestamp" in the code, if it is no longer needed, maybe we can remove the function systemd/systemd.go ?